presentation/frontend/.mocharc.json

```diff
@@ -1,6 +1,7 @@
 {
   "require": [
     "source-map-support/register",
+    "jsdom-global/register",
```

Comment:
Should be switch to `global-jsdom` like in full-stack-tests?

Reply:
maybe in a separate pr?.. this is already way bigger than I'd like it to be